src/marqo/s2_inference/hf_utils.py

```diff
@@ -236,7 +235,7 @@ def extract_huggingface_archive(path: str) -> str:
             try:
                 os.remove(path)
             except Exception as remove_e:
-                raise InvalidModelPropertiesError(
+                raise RuntimeError(
                     f"Marqo encountered an error while attempting to delete corrupted file `{path}`. "
                     f"Please manually check and remove the file. Error message: `{str(remove_e)}`"
                 )
```

Comment:
Just to confirm, what is the error that a user sees (through the REST API) in this situation (perhaps use mocking to simulate this)? 

if you think about this, it is a very bad state to be in (the model is corrupt and Marqo can't remove it).

I think the recommended course of action would be to report the issue and replace the problematic Marqo instance with a new Marqo instance